plugins/modules/terraform.py

```diff
@@ -710,7 +879,9 @@ def process_complex_args(vars):
     if computed_state == 'absent' and workspace != 'default' and purge_workspace is True:
         remove_workspace(terraform_binary, project_path, workspace)
 
-    module.exit_json(changed=plan_file_needs_application, state=computed_state, workspace=workspace, outputs=outputs,
+    diff = dict(before=initial_state_file, after=final_state_file)
```

Comment:
Still need to rename these variables to remove `_file` :)

Reply:
Finally renamed :)